src/connectedk8s/azext_connectedk8s/custom.py

```diff
@@ -947,6 +947,12 @@ def update_connected_cluster(cmd, client, resource_group_name, cluster_name, htt
         patch_cc_response = update_connected_cluster_internal(client, resource_group_name, cluster_name, tags, distribution, distribution_version, azure_hybrid_benefit)
 
     proxy_params_unset = (https_proxy == "" and http_proxy == "" and no_proxy == "" and proxy_cert == "" and not disable_proxy)
+
+    # Returning the ARM response if only AHB is being updated
+    arm_properties_only_ahb_set = (tags is None and distribution is None and distribution_version is None and azure_hybrid_benefit is not None)
+    if proxy_params_unset and auto_upgrade is None and container_log_path is None and arm_properties_only_ahb_set:
+        return patch_cc_response
```

Comment:
Could you add a test case for this change?

Reply:
@zhoxing-ms , The test case for that is already covered in the test file
https://github.com/Azure/azure-cli-extensions/blob/1c98cce60f103a48ca1bb8818208bfd7899155b0/src/connectedk8s/azext_connectedk8s/tests/latest/test_connectedk8s_scenario.py#L50

Reply:
Got it, thanks~